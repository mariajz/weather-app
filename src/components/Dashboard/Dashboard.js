import React from 'react';
import { SafeAreaView } from 'react-native';
import { DashboardWrapper } from './Dashboard.style';
import SearchCitySection from '../search-city-section';

const Dashboard = () => {
    return (
        <SafeAreaView>
            <DashboardWrapper>
                <SearchCitySection />
            </DashboardWrapper>
        </SafeAreaView>
    );
};

export default Dashboard;
