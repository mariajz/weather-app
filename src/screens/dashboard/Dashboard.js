import React from 'react';
import { SafeAreaView } from 'react-native';
import { DashboardWrapper } from './Dashboard.style';
import SearchCitySection from '../../components/search-city-section';
import ForcastSection from '../forcast';

const Dashboard = () => {
    return (
        <SafeAreaView>
            <DashboardWrapper>
                <SearchCitySection />
                <ForcastSection />
            </DashboardWrapper>
        </SafeAreaView>
    );
};

export default Dashboard;
