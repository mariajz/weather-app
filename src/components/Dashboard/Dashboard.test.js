import { render } from '@testing-library/react-native';
import React from 'react';
import Dashboard from './Dashboard';

describe('Dashboard', () => {
    it('should render Dashboard', () => {
        const container = render(<Dashboard />);

        expect(container).toMatchSnapshot();
    });
});
