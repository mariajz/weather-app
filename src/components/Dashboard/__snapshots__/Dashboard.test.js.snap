// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render Dashboard 1`] = `
<RCTSafeAreaView>
  <View
    style={
      {
        "posiiton": "relative",
      }
    }
  >
    <View />
  </View>
</RCTSafeAreaView>
`;
